qiskit/circuit/library/boolean_logic.py

```diff
@@ -0,0 +1,118 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+# pylint: disable=no-member
+
+"""Implementations of boolean logic quantum circuits.
+"""
+
+from typing import List, Optional
+
+import numpy as np
+from qiskit.circuit import QuantumRegister, QuantumCircuit
+from qiskit.circuit.exceptions import CircuitError
+
+
+class Permutation(QuantumCircuit):
+    """An n_qubit circuit that permutes qubits."""
```

Comment:
```suggestion
    """A n_qubit circuit that permutes qubits."""
```

Reply:
I think "an" is correct, see: 
https://www.quora.com/Why-is-an-used-instead-of-a-before-the-word-NGO-even-though-N-is-not-a-vowel

Reply:
Oh! I didn't expect that, sorry for that 🙂